build.gradle

```diff
@@ -53,6 +53,7 @@ dependencies {
     implementation 'io.gsonfire:gson-fire:1.8.5'
     implementation 'org.openapitools:jackson-databind-nullable:0.2.6'
     implementation 'com.google.protobuf:protobuf-java:3.25.2'
+    implementation 'org.junit.jupiter:junit-jupiter-api:5.8.0'
```

Comment:
if assertEquals was the only use case, we can remove this

Reply:
But then how do we use `assertEquals` in the `Pinecone` class?

Reply:
Done!